main/browser-config.ts

```diff
@@ -69,9 +69,10 @@ export default class BrowserConfig {
         if (this.window_state === null) {
             return null;
         }
-        win.once('closed', () => {
+        win.on('resize', () => {
```

Comment:
Yes, but if you access the window state in `'close'` event, you will receive `Object has been destroyed` because the window state have been destroyed in `'close'` event.
If you don't like to save the window size for each `'resize'` event, you can cache the window state in `'resize'` event and save it in the `close` event. This method requires some internal API change and I will leave the decision to you.